__tests__/reducers/literal.test.js

```diff
@@ -23,6 +23,81 @@ describe('literal reducer functions', () => {
     })
   })
 
+  it('SET_ITEMS adds item to an empty state', () => {
+    const emptyState = setMyItems({},
+     {
+       type: 'SET_ITEMS',
+       payload: {
+         rtId: 'resourceTemplate:Monograph:Instance',
+         uri: 'http://schema.org/name',
+         reduxPath: ['resourceTemplate:Monograph:Instance', 'http://schema.org/name'],
+         items: [ { id: 0, content: 'Run the tests'} ]
+       }
+     })
+     expect(emptyState).toEqual({
+       "resourceTemplate:Monograph:Instance": {
+         'http://schema.org/name': {
+           items: [{ id: 0, content: 'Run the tests'}]
+         }
+       }
+     })
+  })
+
+  it('SET_ITEMS adds item to populated state', () => {
+    expect(
+      setMyItems({ "resourceTemplate:Book": {} },
+      {
+        type: 'SET_ITEMS',
+        payload: {
+          rtId: "resourceTemplate:Monograph:Instance",
+          uri: 'http://schema.org/description',
+          reduxPath: ['resourceTemplate:Monograph:Instance', 'http://schema.org/description'],
+          items: [ { id: 2, content: "add this!"}]
+        }
+      })
+    ).toEqual({
+       "resourceTemplate:Book": {},
+      'resourceTemplate:Monograph:Instance': {
+        "http://schema.org/description": {
+          items: [{ id: 2, content: "add this!"}]}
+      }
+    })
+  })
+
+  it('SET_ITEMS creates intermediate objects in state if present in reduxPath', () => {
```

Comment:
I don't understand this description, but maybe that's ok.

Reply:
Since we're not actually initializing the Redux state for embedded resource templates until the `PropertyTemplateOutline` is expanded and data is entered in an included component like InputLiteral, we need to create these intermediate objects if they don't exist.